src/core/Akka.API.Tests/CoreAPISpec.ApproveCluster.approved.txt

```diff
@@ -1,4 +1,5 @@
-﻿[assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute("Akka.Cluster.Metrics")]
+﻿[assembly: System.Reflection.AssemblyMetadataAttribute("RepositoryUrl", "https://github.com/eaba/akka.net")]
```

Comment:
how did this get added to all of these libraries?

Reply:
That line points to my fork of akka.net. If this is not accepted, how can I prevent the tool from adding those?

Reply:
@Aaronontheweb this is what I have discovered:
```csharp
using System;
using System.Reflection;

[assembly: System.Reflection.AssemblyCompanyAttribute("Akka.NET Team")]
[assembly: System.Reflection.AssemblyConfigurationAttribute("Debug")]
[assembly: System.Reflection.AssemblyCopyrightAttribute("Copyright © 2013-2020 Akka.NET Team")]
[assembly: System.Reflection.AssemblyDescriptionAttribute("Replicated data using CRDT structures")]
[assembly: System.Reflection.AssemblyFileVersionAttribute("1.4.12.0")]
[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.4.12+82fbb0e75fe0edc927eba6d9df2004ab04da8cb3")]
[assembly: System.Reflection.AssemblyProductAttribute("Akka.DistributedData")]
[assembly: System.Reflection.AssemblyTitleAttribute("Akka.DistributedData")]
[assembly: System.Reflection.AssemblyVersionAttribute("1.4.12.0")]
[assembly: System.Reflection.AssemblyMetadataAttribute("RepositoryUrl", "https://github.com/eaba/akka.net")]
```  
The last line is a pointer to why that is being added

Reply:
![Screenshot_2020-11-14 Make DistinctRetainingFanOutPublisher a public servant by eaba · Pull Request #4630 · akkadotnet akka](https://user-images.githubusercontent.com/8049896/99146988-61e38400-267d-11eb-9b39-d36ec9464559.png)

Those tests passed locally!

Reply:
your development environment is adding this property to the compiled metadata for the project - but that isn’t happening on the build server. Is there some VS plugin that can add this field? What version of Visual Studio are you using?

Reply:
Whoops, never mind - those tests are racy. Your PR looks good!

Reply:
Visual Studio 16.8
16.8.1 just got released